sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py

```diff
@@ -270,6 +293,227 @@ def create_experiment(
 ]
 
 
+# Mock classes for testing
+class MockTransformerType:
+    __name__ = "MockTransformerType"
+
+
+class HuggingFaceModelParams:
+    def __init__(
+        self,
+        model_uri=None,
+        transformer_type=MockTransformerType,
+        access_token=None,
+        num_labels=None,
+    ):
+        self.model_uri = model_uri
+        self.transformer_type = transformer_type
+        self.access_token = access_token
+        self.num_labels = num_labels
+
+
+class HuggingFaceDatasetParams:
+    def __init__(self, repo_id=None, access_token=None, split=None):
+        self.repo_id = repo_id
+        self.access_token = access_token
+        self.split = split
+
+
+class HuggingFaceTrainerParams:
+    def __init__(self, training_parameters=None, lora_config=None):
+        self.training_parameters = training_parameters
+        self.lora_config = lora_config
+
+
+class S3DatasetParams:
+    def __init__(
+        self,
+        endpoint_url=None,
+        bucket_name=None,
+        file_key=None,
+        region_name=None,
+        access_key=None,
+        secret_key=None,
+    ):
+        self.endpoint_url = endpoint_url
+        self.bucket_name = bucket_name
+        self.file_key = file_key
+        self.region_name = region_name
+        self.access_key = access_key
+        self.secret_key = secret_key
+
+
+class KubeflowOrgV1PyTorchJobSpec:
+    def __init__(
+        self,
+        elastic_policy=None,
+        nproc_per_node=None,
+        pytorch_replica_specs={},
+        run_policy=None,
+    ):
+        self.elastic_policy = elastic_policy
+        self.nproc_per_node = nproc_per_node
+        self.pytorch_replica_specs = pytorch_replica_specs
+        self.run_policy = run_policy
+
+
+class KubeflowOrgV1PyTorchJob:
+    def __init__(
+        self,
+        api_version=None,
+        kind=None,
+        metadata=None,
+        spec=KubeflowOrgV1PyTorchJobSpec,
+        status=None,
+    ):
+        self.api_version = api_version
+        self.kind = kind
+        self.metadata = metadata
+        self.spec = spec
+        self.status = status
+
+
+test_tune_data = [
+    (
+        "not specify name",
+        {
+            "name": None,
+            "objective": lambda x: x,
+            "parameters": {"param": "value"},
+        },
+        ValueError,
+    ),
+    (
+        "set both options",
```

Comment:
I guess, it would be better if you could explain what this error belongs to. Like `invalid hybrid parameters - objective and model_provider_paramters`.